Makefile

```diff
@@ -207,6 +207,7 @@ qgc_firmware: \
 	check_mindpx-v2_default \
 	check_px4fmu-v1_default \
 	check_px4fmu-v2_default \
+	check_px4fmu-v2_lpe \
```

Comment:
OK cool, so that should mean it should be accessible for QGC if we give it the correct url?